### PR TITLE
Support ES 7.1.1

### PR DESCRIPTION
**Pull request by seraphjiang**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by seraphjiang** [comment]
Test Suites: 6 skipped, 70 passed, 70 of 76 total
Tests:       20 skipped, 341 passed, 361 total
Snapshots:   121 passed, 121 total
Time:        36.023s
Ran all test suites.
✨  Done in 38.16s.

**Comment by seraphjiang** [comment]
Tested with opendistro_alerting-1.1.0.0-SNAPSHOT

File: opendistro-elasticsearch-alerting-kibana.release-notes.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
-## Version 0.8.0 (Current)
+## Version 1.1.0 (Current)
```
Comment (mihirsoni) [inline opendistro-elasticsearch-alerting-kibana.release-notes.md]: It should be 1.1.0.0

File: opendistro-elasticsearch-alerting-kibana.release-notes.md [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+
+### New Features
+ * Add throttle [Alerting #14](https://github.com/opendistro-for-elasticsearch/alerting/issues/14)
+ * Add support for Kibana 7.1.1
+
+ ## Bug fixes
+ * Set min property on interval input[#19](https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/issues/19)
+ * No documents for aggregations count as false [#22](https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/issues/22)
+ * Add DELETED alert state to Dashboard filter[#30](https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/issues/30)
+ * Enhanced message for trigger execution [Alerting #58](https://github.com/opendistro-for-elasticsearch/alerting/issues/14)
```
Comment (mihirsoni) [inline opendistro-elasticsearch-alerting-kibana.release-notes.md]: This should be part of new feature.

**Pull request merged**